container/CrabV2/Auction/Admin/AdminBidView.tsx

```diff
@@ -407,6 +393,62 @@ const AdminBidView: React.FC = () => {
     await submitTx(tx.hash, tx.timestamp || 0)
   }
 
+  const executeBullAuction = async () => {
+    try {
+      const { orders } = getOrders()
+      const availableOsqth = orders.reduce((acc, o) => acc.add(o.quantity), BIG_ZERO)
+      const auctionSqth = BigNumber.from(auction.oSqthAmount)
+      const { oSQTHAuctionAmount: neededSqth } = await getBullAuctionDetails()
+
```

Comment:
Why do we care about getBullAuctionDetails has to say at this point?

Say the auction runs for auctionSqth = 1000 and gets availableOsqth = 1000 in orders, but getBullAuctionDetails returns neededSqth = 900 because the prices have changed. Then auctionOsqthAmount = 900 and we only trade 900 contracts.